project/Build.scala

```diff
@@ -984,6 +984,8 @@ object Build {
           ++ (dir / "shared/src/test/scala/org/scalajs/testsuite/javalib/io" ** (("ThrowablesTest.scala": FileFilter) || "SerializableTest.scala" || "PrintWriterTest.scala")).get
           ++ (dir / "shared/src/test/scala/org/scalajs/testsuite/javalib/math" ** "*.scala").get
           ++ (dir / "shared/src/test/scala/org/scalajs/testsuite/javalib/net" ** ("URLDecoderTest.scala": FileFilter)).get
+          ++ (dir / "shared/src/test/scala/org/scalajs/testsuite/javalib/security" ** ("ThrowablesTest.scala": FileFilter)).get
```

Comment:
Prefer using `"*.scala"` when a group is complete, even if it contains only one file. That way, if we later add tests to that group, they will automatically be taken into account if they succeed. And if they fail, we'll add an exception. If they're not enabled by default and they would succeed, we wouldn't notice, and hence lose a testing opportunity.

Reply:
Good point, I fixed it